diff --git a/dist/bun-sqlite-dialect-PuE7pPr-.mjs b/dist/bun-sqlite-dialect-PuE7pPr-.mjs
index 9a4b3dfa7a73b482d6211f661cbaf4a7df3edbf6..bf7352a1dd9589ee72d073613594abb5bfc3b05d 100644
--- a/dist/bun-sqlite-dialect-PuE7pPr-.mjs
+++ b/dist/bun-sqlite-dialect-PuE7pPr-.mjs
@@ -1 +1,3 @@
-import { CompiledQuery, DEFAULT_MIGRATION_LOCK_TABLE, DEFAULT_MIGRATION_TABLE, DefaultQueryCompiler, sql } from "kysely";
+import { CompiledQuery, DefaultQueryCompiler, sql } from "kysely";
+const DEFAULT_MIGRATION_TABLE = "kysely_migration";
+const DEFAULT_MIGRATION_LOCK_TABLE = "kysely_migration_lock";
diff --git a/dist/d1-sqlite-dialect-C2B7YsIT.mjs b/dist/d1-sqlite-dialect-C2B7YsIT.mjs
index 1545e428a73da3747b96b142d7a0f40eaaa41b0e..6e900a2810961832d3e7f2db51e7e7da34c73601 100644
--- a/dist/d1-sqlite-dialect-C2B7YsIT.mjs
+++ b/dist/d1-sqlite-dialect-C2B7YsIT.mjs
@@ -1 +1,3 @@
-import { DEFAULT_MIGRATION_LOCK_TABLE, DEFAULT_MIGRATION_TABLE, SqliteAdapter, SqliteQueryCompiler } from "kysely";
+import { SqliteAdapter, SqliteQueryCompiler } from "kysely";
+const DEFAULT_MIGRATION_TABLE = "kysely_migration";
+const DEFAULT_MIGRATION_LOCK_TABLE = "kysely_migration_lock";
diff --git a/dist/node-sqlite-dialect.mjs b/dist/node-sqlite-dialect.mjs
index 8c836b642cc205f52345c2d86ba80a4bb5123f12..992c66e9312d4cd077c06942eeb66a6e2a886dee 100644
--- a/dist/node-sqlite-dialect.mjs
+++ b/dist/node-sqlite-dialect.mjs
@@ -1 +1,3 @@
-import { CompiledQuery, DEFAULT_MIGRATION_LOCK_TABLE, DEFAULT_MIGRATION_TABLE, DefaultQueryCompiler, sql } from "kysely";
+import { CompiledQuery, DefaultQueryCompiler, sql } from "kysely";
+const DEFAULT_MIGRATION_TABLE = "kysely_migration";
+const DEFAULT_MIGRATION_LOCK_TABLE = "kysely_migration_lock";
